modules/ROOT/pages/about-flows.adoc

```diff
@@ -42,20 +37,27 @@ out to other services, you might factor out that behavior into other flows.
 
 == Using Subflows
 
-You can also group a series of event processors into subflows. Like a flow, a
-subflow is a scope that groups together a sequence of event processors, but it
-does not have an event source, and it does not have its own error handling.
-Subflows are always called from another flow or subflow through a Flow Ref
-component.
+A subflow is a scope that enables you to group event processors, similar to a flow, but with certain differences and limitations:
+
+* Subflows do not have event sources.
+* Subflows do not have an error handling scope.
+* During design, subflows work as a macro that replaces the Flow Reference components that call the subflow. When you build the application, all Flow Reference components that call a subflow are replaced by the contents of the referenced subflow.
+* Referencing a subflow has higher performance than referencing a flow.
+
+=== Considerations
+
+Because the contents of the subflow replace each Flow Reference component that references the subflow, several instances of the event processors inside the subflow exist in the application at runtime. This behavior can cause issues with event processors that depend on unique IDs or instances to execute like Batch processors.
+
+For example, configuring a Batch Job inside a subflow causes the application to fail during deployment if the subflow is referenced from more than one Flow Reference component. The application fails to deploy because multiple instances of a Batch Job with the same Job Instance ID exist, which is not allowed.
```

Comment:
```suggestion
For example, configuring a batch job inside a subflow causes the application to fail during deployment if the subflow is referenced from more than one Flow Reference component. The application fails to deploy because multiple instances of a batch job with the same job instance ID exist, which is not allowed.
```